site/docs/guide/filter-queries.md

```diff
@@ -201,6 +190,93 @@ bot
 The type inference of `ctx` will scan through the entire call chain and inspect every element of all three `.on` calls.
 As an example, it can detect that `ctx.msg.text` is a required property for the above code snippet.
 
+## Useful Tips
+
+Here are some less known features of filter queries that can come in handy.
+Some of them are a little advanced, so feel free to move on to [the next section](./commands.md).
+
+### Chat Member Updates
+
+You can use the following filter query to receive status updates about your bot.
+
+```ts
+bot.on("my_chat_member"); // start, stop, join, or leave
+```
+
+In private chats, this triggers when the bot is started or stopped.
+In groups, this triggers when the bot is added or removed.
+You can now inspect `ctx.myChatMember` to figure out what exactly happened.
+
+This is not to be confused with
+
+```ts
+bot.on("chat_member");
+```
+
+which can be used to detect status changes of other chat members, such as when people join, get promoted, and so on.
```

Comment:
```suggestion
Which can be used to detect status changes of other chat members, such as when people join, get promoted, and so on.
```

Reply:
This would split up the sentence in the middle. If you don't like to use code blocks in sentences (I know you already don't like formatting sentences as lists) then we should just rephrase this paragraph.